docker-compose.yml

```diff
@@ -31,6 +31,6 @@ services:
     environment:
```

Comment:
- use graph node 0.16.1 https://github.com/graphprotocol/graph-node/releases/tag/v0.16.1
- IPFS max file size should be increased to 300000.